src/swatiM/assignments/StringLength.java

```diff
@@ -0,0 +1,57 @@
+// 1. write a method to return length of the given name
+// int getNameLength(String name)
+
+//2. return middle character from given name.
+// Hint : char getMiddleChar(String name)
+
+// input : Maulik
+// output : Middle character in name Maulik is u
+
+// input : Technocredits
+// output : Middle character in name Technocredits is c
+/*
+ * Note : 
+Method name convention:  
+
+1.int getNameLength(String name) : if method is returning any value except boolean, method name should start with get
+2. boolean isPrimeNumber() : if method is returning boolean value, name should start with is
+3. display() : if method is not returning anything, any name would be ok but dont just start it with get or is.
+4. printInfo() : if method is prininting something, we can start method name with print keyword.
+ */
+
+package swatiM.assignments;
+
+public class StringLength {
+
+	int getNameLength(String name) {
+		int length = name.length();
+		return length;
+
+	}
+
+	char getMiddleChar(String name) {
+		int index = 0;
+		int len = name.length();
```

Comment:
instead of calling direct method you should call **getNameLength**()

Reply:
![image](https://user-images.githubusercontent.com/112226778/189104903-0b39af98-b153-4b9e-8c7c-ef1dedba3c33.png)

updated code and commit changes please review